f5_openstack_agent/lbaasv2/drivers/bigip/icontrol_driver.py

```diff
@@ -1098,10 +1118,15 @@ def _update_loadbalancer_status(self, loadbalancer):
 
         if (provisioning_status == plugin_const.PENDING_CREATE or
                 provisioning_status == plugin_const.PENDING_UPDATE):
+            operational_status = lb_const.ONLINE
+            if self.disconnected_service_polling.enabled:
+                # operational status will be set by the disconnected
+                # service polling thread if that mode is enabled
+                operational_status = lb_const.OFFLINE
```

Comment:
I guess this is why the call above works because you set it to OFFLINE by default?